atlassian_jwt_auth/contrib/django_app/decorators.py

```diff
@@ -0,0 +1,41 @@
+from functools import wraps
+from django.http import HttpResponse, HttpResponseForbidden
+from django.conf import settings
+
+import jwt
+
+from atlassian_jwt_auth.contrib.django_app.auth import ASAPAuthBackend
+
+
+class HttpResponseNotAuthorized(HttpResponse):
+    status_code = 401
+
+
+def asap_required(view_func, issuers=None):
+    """Decorator which ensures the user has provided a correct user and token pair."""
+
+    @wraps(view_func)
+    def _wrapped_view(request, *args, **kwargs):
+        user = request.user
+
```

Comment:
In this case, the blank lines improve readability of the function and provide clear separation between the docstring and the nested function. This is not an abnormal styling in most Python projects.